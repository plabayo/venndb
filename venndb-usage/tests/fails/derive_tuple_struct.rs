use venndb::VennDB;

#[derive(VennDB)]
struct MyStruct(u32);

fn main() {}
