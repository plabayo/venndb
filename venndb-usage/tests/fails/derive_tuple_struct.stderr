error: `#![derive(VennDB)]` is not currently supported on tuple structs
 --> tests/fails/derive_tuple_struct.rs:4:1
  |
4 | struct MyStruct(u32);
  | ^^^^^^
